components/Accordion/src/index.tsx

```diff
@@ -1,4 +1,6 @@
 import Accordion from './Accordion';
+import './index.css';
+import './stories.js';
```

Comment:
This would probably result in stories.js being added to the build and NPM packages. This script is only for use in storybook right? In that case we should probably import it in preview.js or another storybook-only file.